source/release-notes/release-4-9-2.rst

```diff
@@ -0,0 +1,44 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+   :description: Wazuh 4.9.2 has been released. Check out our release notes to discover the changes and additions of this release.
+
+4.9.2 Release notes - TBD
+=========================
+
+This section lists the changes in version 4.9.2. Every update of the Wazuh solution is cumulative and includes all enhancements and fixes from previous releases.
+
+What's new
+----------
+
+This release includes new features or enhancements as the following:
+
+Resolved issues
+---------------
+
+This release resolves known issues as the following:
+
+Changelogs
+----------
+
+The repository changelogs provide more details about the changes.
+
+Product repositories
+^^^^^^^^^^^^^^^^^^^^
+
+-  `wazuh/wazuh <https://github.com/wazuh/wazuh/blob/v4.9.2/CHANGELOG.md>`__
+-  `wazuh/wazuh-dashboard-plugins <https://github.com/wazuh/wazuh-dashboard-plugins/blob/v4.9.2-2.11.0/CHANGELOG.md>`__
```

Comment:
```suggestion
-  `wazuh/wazuh-dashboard-plugins <https://github.com/wazuh/wazuh-dashboard-plugins/blob/v4.9.2/CHANGELOG.md>`__
```

Reply:
Done in [8c4e18f](https://github.com/wazuh/wazuh-documentation/pull/7742/commits/8c4e18ff557062cad97e924419c1ab6ea61341bf)